.travis.yml

```diff
@@ -23,11 +25,18 @@ matrix:
       env: SYMFONY_VERSION="2.7.*"
     - php: 5.6
       env: SYMFONY_VERSION="2.8.*" DEPENDENCIES=dev
+    - php: 7.1
+      env: SYMFONY_VERSION="2.8.*" DEPENDENCIES=dev
+    - php: 7.2
+      env: SYMFONY_VERSION="2.8.*" DEPENDENCIES=dev
```

Comment:
there is no reason to test the Symfony 2.8 LTS on multiple PHP versions. Please remove these 2 jobs